_layouts/home.html

```diff
@@ -1,70 +1,295 @@
+            <div class="ps-0 ps-md-3 mt-3 mt-md-0">
+              <div>
+                <div id="legend-c"></div>
+                <div>
+                  <canvas id="chart-c"></canvas>
+                </div>
               </div>
             </div>
-            <div class="col-md-6">
-              <div class="info">
-                <i class="material-icons text-gradient text-primary text-3xl">flip_to_front</i>
-                <h5 class="font-weight-bolder mt-3">Bootstrap 5 Ready</h5>
-                <p class="pe-3">The world’s most popular front-end open source toolkit, featuring Sass variables and
-                  mixins.</p>
+          </div>
+        </div>
+        <script>
+          const DATA_A = {
+            "Technical skills (general)": {
+              "Collaborative software development": 3,
+              "Version control": 3,
+              "Continuous integration / continuous delivery": 2,
+              "Systems design, architecture & development": 4,
+              "Software documentation": 5,
+              "Testing": 1,
+              "Coding conventions": 2,
+              "Software licensing": 1,
+              "Software packaging & release": 3,
+              "Programming paradigms": 1,
+              "Programming": 4,
+              "Environmental impact awareness": 3
+            },
+            "Technical (specific)": {
+              "Visualization": 2,
+              "High performance computing": 4,
+              "Programming languages": 1,
+              "Parallelization": 4,
+              "Web development": 7,
+              "Systems administration": 1,
+              "Data analysis": 1,
+              "AI/ML": 7,
+              "User Experience": 3,
+              "Technical translation between researchers and RSEs": 1,
+              "Virtualization & Cloud computing": 4,
+              "Code optimization": 2
+            },
+            "Interpersonal": {
+              "Understanding of mental health, stress and burnout": 6,
+              "Building of community of practice": 3,
+              "Collaboration & building connections": 4,
+              "People management": 4,
+              "Leadership & communication skills": 1,
+              "Mentorship skills": 4,
+              "Stakeholder management": 4,
+              "Diversity, Equity and Inclusion": 4
+            },
+            "Leadership & management": {
+              "Project management": 7,
+              "Planning and organising events (workshops, meet-ups, conferences etc)": 1,
+              "Grant/proposal writing": 4,
+              "Evaluation and assessment": 1,
+              "Agile project management methodologies": 5,
+              "Recruitment": 4,
+              "Resource management (time, budget)": 7,
+              "Strategy": 4,
+              "Change management": 3,
+              "Domain expertise": 3,
+              "Research methodologies": 1,
```

Comment:
The domain expertise category isn't formatted right here.
```suggestion
              "Change management": 3
            },
            "Domain expertise": {
              "Research methodologies": 1,
```